library/psa_crypto.c

```diff
@@ -93,10 +93,25 @@ static int key_type_is_raw_bytes(psa_key_type_t type)
 #define RNG_INITIALIZED 1
 #define RNG_SEEDED 2
 
+typedef enum {
+    PSA_CRYPTO_SUBSYSTEM_DRIVER_WRAPPERS = 0,
```

Comment:
Nitpick: Starting the values at 1 would be more robust as we would catch passing uninitialised selectors (if the compiler zeroes them out).